spread/qemu.go

```diff
@@ -114,10 +116,18 @@ func (p *qemuProvider) Allocate(ctx context.Context, system *System) (Server, er
 	serial := fmt.Sprintf("telnet:127.0.0.1:%d,server,nowait", port+100)
 	monitor := fmt.Sprintf("telnet:127.0.0.1:%d,server,nowait", port+200)
 	fwd := fmt.Sprintf("user,hostfwd=tcp:127.0.0.1:%d-:22", port)
-	cmd := exec.Command("qemu-system-x86_64", "-enable-kvm", "-snapshot", "-m", strconv.Itoa(mem), "-net", "nic", "-net", fwd, "-serial", serial, "-monitor", monitor, path)
+	cmd := exec.Command(qemuBinary, "-enable-kvm", "-snapshot", "-m", strconv.Itoa(mem), "-net", "nic", "-net", fwd, "-serial", serial, "-monitor", monitor, path)
 	if os.Getenv("SPREAD_QEMU_GUI") != "1" {
 		cmd.Args = append([]string{cmd.Args[0], "-nographic"}, cmd.Args[1:]...)
 	}
+	switch system.Bios {
+	case "legacy", "":
+		// nothing to do, that is the qemu default
```

Comment:
I would leave a TODO to warn about empty value some point.

Reply:
I like the idea, if I understand you correctly it requires that I'm able to differentiate between unset and set to the empty string which I don't think I can do currently. It would require something similar to "OptionalInt" in project.go. Not a big deal but I would not want to make this PR bigger and harder to review just for this.

Reply:
All you need is a pointer to a string in the struct. Then you can do system.Bios == nil vs *system.Bios == "legacy"

Reply:
Let's do this later/in a followup, I want this initial version to be as small and reviewable as possible.